what-we-do.markdown

```diff
@@ -27,7 +27,7 @@ What-section-one-description: "Tiko connects across supply and demand, building
   and we boost local economies wherever we go. Our signature ecosystem approach means
   we leverage expertise, funds and infrastructure to scale fast. \n\nOur ecosystems
   are at varying stages of maturity from nascent projects in Burkino Faso to established
-  programmes in India. In Kenya for example, we are currently operating 145 such ecosystems
+  programmes in Kenya. In Kenya for example, we are currently operating 145 such ecosystems
```

Comment:
Is the information still true given the change to Kenya?

Reply:
Hmm, good question.

The restructure of the sentence is fine given the context, but I'm not sure about the 145 figure in the next sentence.

I'll follow up with Tam.

Thanks Bady!